### ISSUE TITLE
Can't download files

### ISSUE THREAD

**Issue by Deksor** [jhhoward/MicroWeb#8]
Hello, very nice project. However I think you should check for the mime type of the HTTP response before trying to parse the html, and if it's not HTML, you could let people download the file :)

**Comment by jhhoward** [comment]
Yes I would like to add this feature, ideally as a background process so that the user can continue to browse other pages

**Comment by BigMonstro** [comment]
Hello jhhoward.

Starting a such development in 2021 is AWESOME !

I am the current administrator of a website dedicated to retrocomputing (win3x.org : it hosts a forum, many programs for old computers and an improved version of total hardware 99 : http://www.win3x.org/uh19) and I have to say I was pleasantly surprised when I discovered the existence of your browser.

Microweb works fine and it's blazing fast on my 486DX2 (I've immediately noticed it because the speed sharply contrasts with so many browsers, even those released in the late 90's. Sure, it's much stronger than the targeted systems: 8088 class computers).

Of course, the browser has few abilities ; that's understandable, not only because it's a very recent project but also because it's intended for incredibly limited machines. Yet, I like it, as it's a functional HTML2 browser that is lightweight enough to be stored on a floppy disk. With some additions, it could become a masterpiece for many retro enthusiasts like me :)

I don't think adding the support of css, https, images, javascript or tables would be realistic, considering the goal of that browser is being useable with the slowest x86 processor ever made. Without EMS/XMS, memory is also a critical limitation.

However, I strongly supports the Deksor's proposition. After all, although the very earliest PCs didn't have a hard drive, it became quite common after 1983, with the release of IBM PC XT. Windows 2.1, released in 1988, requires a mass storage unit but the /286 version remains compatible with the 8088 processor, it was the norm at the time. You can download small programs from a 8088 by using mTCP on ftp://37.187.103.91. You can even extract zip files thanks to Pkzip 2.04g. Therefore, it would be great if Microweb will be able to download programs from http://37.187.103.91/files or even from the website http://www.win3x.org !

Best regards,
Big Monstro
